lib/react.dart

```diff
@@ -65,6 +65,38 @@ typedef ReactDartFunctionComponentFactoryProxy FunctionComponentRegistrar(DartFu
 /// See: <https://reactjs.org/docs/fragments.html>
 var Fragment = ReactJsComponentFactoryProxy(React.Fragment);
 
+/// [Suspense] lets you display a fallback UI until its children have finished loading.
+///
+/// Like [react.Fragment], [Suspense] does not render any visible UI.
+/// It lets you specify a loading indicator in case some components in
+/// the tree below it are not yet ready to render.
+/// [Suspense] currently works with:
+/// - Components that use React.lazy
+///   - (dynamic imports, not currently implemented in dart)
```

Comment:
Should this be a nested indent?

```suggestion
/// - (dynamic imports, not currently implemented in dart)
```

Reply:
that was intentional as it is a clarification on the point above it.